packages/components/src/components/chat-search/ContextStatus.vue

```diff
@@ -0,0 +1,209 @@
+<template>
+  <v-status-bar v-if="appmapStats && appmapStats.numAppMaps" level="info">
```

Comment:
`appmapStats` will be updated soon to be [`Stats | Stats[]`](https://github.com/getappmap/appmap-js/blob/350c84aa8fa5b5282224e68a6ab7291b73b070d0/packages/rpc/src/appmap.ts#L10-L19). It's probably worth adding support for this.

Reply:
@dustinbyrne is this the time to introduce the "front end version" flag and have the backend be responsive to this flag? This way, the frontend can declare what features it can handle. In this case, `Stats` would be the old behavior and `Stats[]` (even if it's an array with 1 element) would be the new behavior.

Reply:
I'm going to add this to #1625